packages/rps/contracts/RockPaperScissors.sol

```diff
@@ -116,7 +116,9 @@ contract RockPaperScissors is ForceMoveApp {
         private
         pure
         outcomeUnchanged(fromPart, toPart)
-        stakeUnchanged(fromGameData, toGameData)
+        // TODO: question (by @liam to @george) this is wrong right? the stake _will_ change
+        // because we are proposing a new round, so this line of code below should not be here?
+        // stakeUnchanged(fromGameData, toGameData)
```

Comment:
Question for @geoknee

Reply:
I think the stake could be allowed to change, so agree that arguably this modifier does not belong here. 

I would not go so far as to say the stake _will_ change. In our current app implementation, for example, it certainly will _not_ change. 

In fact it seems reasonable (if unnecessary) to bake this into the rules by leaving this modifier in place.